opm/parser/eclipse/EclipseState/Runspec.hpp

```diff
@@ -53,19 +53,57 @@ class Phases {
 };
 
 
+/// Subset of WELLDIMS keyword to answer size questions for restart
+/// purposes.
+class Welldims {
+public:
+    /// Constructor
+    explicit Welldims(const Deck& deck);
+
+    /// Retrieve maximum number of connections/perforations per well.
+    int maxConnPerWell() const
+    {
+        return this->nCWMax;
+    }
+
+    /// Retrieve maximum number of wells or sub-groups per well group.
```

Comment:
OK - I realize we might disagree on this; but to me such comments are noise. The comments in this PR - except the one regarding the difference between e100 and e300 are all *noise* - I would prefer if they were removed.

Reply:
> OK - I realize we might disagree on this; but to me such comments are noise.

I tend to think they're noise too.  I just wasn't clear on the project's policy for Doxygen-style reference documentation.  I'll happily remove them (except possibly for the data members).